package.json

```diff
@@ -39,6 +39,9 @@
     "@emotion/styled": "^10.0.27",
     "@govuk-react/icon-crown": "^0.0.5",
     "@govuk-react/icons": "^0.7.1",
+    "@slice-and-dice/govuk-react": "^0.7.13",
+    "@slice-and-dice/govuk-react-button": "^0.7.2",
+    "@slice-and-dice/govuk-react-radio": "^0.7.2",
```

Comment:
Is there a PR for these changes in the govuk-react repo?
Should be PR from `**slice-and-dice-packages/<feature>**` to `**slice-and-dice-packages/base**`